pkg/operator/controller/ca-configmap/controller.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ca_configmap
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	extdnscontroller "github.com/openshift/external-dns-operator/pkg/operator/controller"
+	ctrlutils "github.com/openshift/external-dns-operator/pkg/operator/controller/utils"
+)
+
+const (
+	controllerName = "ca_configmap_controller"
+)
+
+// Config holds all the things necessary for the controller to run.
+type Config struct {
+	SourceNamespace string
+	TargetNamespace string
+	CAConfigMapName string
+}
+
+type reconciler struct {
+	scheme *runtime.Scheme
```

Comment:
Come to think of it, we don't use `scheme` either, do we?

Reply:
you are right, no, we don't. removed it.